registry/quilt_server/auth.py

```diff
@@ -272,8 +272,9 @@ def revoke_user_code_tokens(user):
         db.session.delete(code)
     revoke_tokens(user)
 
-def get_exp(mins=30):
-    return datetime.utcnow() + timedelta(minutes=mins)
+def get_exp(mins=None):
+    delta = timedelta(minutes=mins) if mins is not None else timedelta(days=90)
```

Comment:
this is funky. either don't take` minutes` as keyword arg or take both `minutes` and `days` and pass all of them on to `timedelta`. i'm guessing you're aiming for backwards compatibility, but i don't think it's worth it given how confusing this is.

atlernatively, make `mins=60*24*30` the default. and that brings me to another issue, `mins=None` is not a very useful default.

Reply:
you could also take `**kwargs` and pass it through to `timedelta`; just put that in the docstring